docs/marionette.behavior.md

```diff
@@ -306,6 +308,9 @@ triggered on a `View` are passed to all attached `behaviors`. This includes:
 These handlers work exactly as they do on `View` -
 [see the `View` documentation](./marionette.view.md#events)
 
+> Events triggered in behavior instance are not executed in view. To notify 
```

Comment:
can we say "in the behavior instance" and "in the view" to match the next sentence?